card/spec/lib/card/view_cache_spec.rb

```diff
@@ -1,3 +1,5 @@
+# -*- encoding : utf-8 -*-
```

Comment:
no objection to these, but they only matter if there are non-ascii characters in the code, right?